fanuc_lrmate200id_gazebo/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(fanuc_lrmate200id_gazebo)
+find_package(catkin REQUIRED)
+catkin_package()
```

Comment:
Could you please add the required `install()` commands here? See [fanuc_m10ia_support/CMakeLists.txt](https://github.com/ros-industrial/fanuc/blob/e632abcb4c8f223879884da56fe412a6d488066a/fanuc_m10ia_support/CMakeLists.txt#L15-L16) for an example.